modules/gapi/test/common/gapi_tests_common.hpp

```diff
@@ -334,7 +334,7 @@ class TestFunctional
 #endif
     // initialize std::vector<cv::Point_<T>>/std::vector<cv::Point3_<T>>
     template <typename T, template <typename> class Pt>
-    void initPointsVectorRandU(const int sz_in, std::vector<Pt<T>> &vec_)
+    void initPointsVectorRandU(const int sz_in, std::vector<Pt<T>> &vec_) const
```

Comment:
Why did you add the const?

Reply:
It could be needed in the previous versions of the code, if I wanted to pass the const reference of the test object to a function and initialize a vector inside it.

Now it isn't actually used, but I decided to preserve this change as in fact this method is `const` because it doesn't change the class's fields. So, I made a mistake not to add the `const` when I wrote this method in the past and want to correct it here.